content/articles/close-account.md

```diff
@@ -19,16 +19,25 @@ To delete yourself as a user, you'll first need to [unsubscribe your accounts fr
 <div class="section-steps" markdown="1">
 ##### To permanently delete yourself as a user, along with all your domains and records:
 
+1. [Unsubscribe your accounts from your plan(s) at DNSimple](/articles/cancel-subscription).
+
 1. Go to [your User Settings page](https://dnsimple.com/user):
 
-    ![Navigate to user settings page](/files/access-user-settings.jpg)
+    **New screenshot** 
```

Comment:
https://drive.google.com/file/d/1Lxuvl7kHKDeaQtdbx7GTn6cam8TBq2pM/view?usp=sharing

Reply:
@dperrymorrow These screenshots were not added. Please reopen and add the screenshots.